.vscode/settings.json

```diff
@@ -8,6 +8,6 @@
     "eslint.autoFixOnSave": true,
     "editor.tabSize": 2,
     "editor.codeActionsOnSave": {
-        "source.fixAll.eslint": true
+        "source.fixAll.eslint": "explicit"
```

Comment:
Guessing this file was accidentally committed

```suggestion
        "source.fixAll.eslint": true
```

Reply:
Whoops! Yep, unintentional. Thanks!